synfig-studio/src/gui/widgets/widget_timeslider.cpp

```diff
@@ -303,23 +309,52 @@ Widget_Timeslider::on_draw(const Cairo::RefPtr<Cairo::Context> &cr)
 				double x1 = time_plot_data->get_double_pixel_t_coord(bounds[i][1]);
 				double w = x1 - x0;
 
+				double boundary_dimension=8;
```

Comment:
`const`?
spaces surrounding assignment operator `=`

Reply:
sure const seems reasonable.

Reply:
actually here I switch up boundary_dimension value depending on lower/upper bound so I think maybe better to name it `boundary_adjust `and have another const for the actual dimension... the 8 in this case